pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -64,9 +65,10 @@ func (mixerplugin) OnOutboundListener(in *plugin.InputParams, mutable *plugin.Mu
 	}
 
 	attrs := attributes{
-		"source.uid":             attrUID(in.Node),
-		"context.reporter.uid":   attrUID(in.Node),
-		"context.reporter.local": attrBoolValue(false),
```

Comment:
can we kill this straight away? I feel like we have config using it that will stop working correctly if it is removed altogether. including `stackdriver`.  should we maintain it until that config is transitioned?

cc: @bianpengyuan

Reply:
I'd prefer killing it straight away and updating the dependencies. We have not advertised it that much. I found only one use in a proto comment (that I failed to update since make generate has a huge effect on my machine).

Reply:
did you try using `context.reporter.type` in the expression language?  Because I'm trying it now and I think the fact that `type` is a reserved word causes all sorts of issues.

```
failed to evaluate expression for field 'Dimensions[reporter]'; failed to parse expression 'context.reporter.type"': unable to parse expression 'context.reporter.type': ... expected selector or type assertion, found 'type'
```